Cannot find module './transformers/module'

I get this while building docs:

```
[vite-plugin-svelte] Error while preprocessing /tmp/tmp5akpx6mx/kit/src/routes/index.mdx - Cannot find module './transformers/module'
```

https://github.com/huggingface/huggingface.js/actions/runs/4472632477/jobs/7859025182

--- a/kit/svelte.config.js
+++ b/kit/svelte.config.js
@@ -20,7 +20,11 @@ const config = {
 		inferenceSnippetPreprocess,
 		tokenizersLangPreprocess,
 		mdsvexPreprocess,
-		preprocess({ sourceMap: Boolean(process.env.DOCS_SOURCEMAP) })
+		preprocess({ 
+			sourceMap: Boolean(process.env.DOCS_SOURCEMAP),
+			// So there can be code snippets with <script type="module"> in the docs
+			preserve: ["module"]
+		})
 	],
 
 	kit: {

